package.json

```diff
@@ -164,7 +164,7 @@
     "copy-wasm": "rm -f dist/flux-lsp-node_bg.wasm && cp node_modules/@influxdata/flux-lsp-node/flux-lsp-node_bg.wasm dist/",
     "vscode:prepublish": "npm run webpack-prod && npm run copy-wasm",
     "webpack-dev": "webpack --mode development --watch",
-    "webpack-prod": "webpack --mode production",
+    "webpack-prod": "webpack --mode development",
```

Comment:
Can we change this back to production mode?

Reply:
Oops!